Apps/backend/src/main.rs

```diff
@@ -1,15 +1,29 @@
 use std::{env};
```

Comment:
Take care when merging.
The rust "backend" has been renamed to "generator" and the middleware node server has been moved out of frontend into "backend".